packages/core/src/App/Containers/Layout/header/dtrader-header.tsx

```diff
@@ -86,32 +82,32 @@ const DTraderHeader = observer(() => {
         >
             <div className='header__menu-items'>
                 <div className='header__menu-left'>
-                    {!is_mobile && (
+                    {isDesktop && (
                         <PlatformSwitcher
                             app_routing_history={app_routing_history}
                             platform_config={filterPlatformsForClients(platform_config)}
                             setTogglePlatformType={setTogglePlatformType}
                             current_language={current_language}
                         />
                     )}
-                    {is_mobile && (
+                    {!isDesktop && (
```

Comment:
is it possible to unite isDesktop & !isDesktop into a ternary here? :)
isDesktop ? ... : ...